easybuild/easyblocks/c/clang.py

```diff
@@ -113,6 +116,29 @@ def __init__(self, *args, **kwargs):
         self.llvm_obj_dir_stage3 = None
         self.make_parallel_opts = ""
 
+        if not self.cfg['llvm_projects']:
+            self.cfg['llvm_projects'] = []
+
+        # keep compatibility between using llvm_projects vs using flags
```

Comment:
It is confusing that flags can overwrite `llvm_projects` in version >=14 but is the opposite for versions <14. Can we consolidate these settings in a common variable and use that single source of settings in the rest of the easyblock? For instance, I think we can always use `llvm_projects` and replace checks for versions <14 such as `if self.cfg["usepolly"]` for `if "polly" in llvm_projects`.

Reply:
I wanted to minimize changes, but sure, I can do that. For some however, the condition is trickier.

Reply:
addressed in ae86d8876